types/experiments.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * The experiment definition accessible to Firefox via Remote Settings.
+ * It is compatible with ExperimentManager.
+ */
+export interface ExperimentRecipe {
+  /** A unique identifier for the Recipe */
+  id: string;
+  /** JEXL expression defined in an Audience */
+  filter_expression: string;
```

Comment:
There was also a fun discussion in https://github.com/mozilla/nimbus-shared/pull/19#discussion_r439678347, where we discussed possibly adding the ESLint `camelcase` rule to enforce that properties should always be camelcase, except where they are known exceptions. Not copying the entire thread here since I got silly with ESLint rule configs and outputs, but it's in the linked discussion for anybody reviewing this (if ESLint is your thing).